package.json

```diff
@@ -49,9 +40,10 @@
     "webpack": "^3.6.0",
     "webpack-node-externals": "^1.6.0",
     "window-size": "^1.1.0",
-    "wrap-ansi": "^3.0.1"
+    "wrap-ansi": "^3.0.1",
+    "zlib": "^1.0.5"
   },
   "dependencies": {
-    "measurement-kit": "^0.1.0-alpha.5"
+    "measurement-kit": "^0.1.0-alpha.8"
```

Comment:
Thing that wonders me: the versioning of Node bindings is not aligned with the version of MK. Do you think that this may be a problem? FTR for Android we use `${mk_version}-${bindings-version}`.